src/features/common/Layout/ProjectsLayout/ProjectsLayout.module.scss

```diff
@@ -31,3 +31,36 @@
   border-radius: 16px;
   overflow: hidden;
 }
+
+// styles for mobile layout
+
+.mobileProjectsLayout {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  padding: 100px 5px 20px;
```

Comment:
1. Use margin for the top spacing instead of padding. You can use `height: calc(100vh - 100px)`. Use padding to describe the visible space around the box - it makes it easier to understand/debug the layout
2. Padding on the sides should be 0, as some elements can take full width